tests/integration/high_availability/test_upgrade_rollback_incompat.py

```diff
@@ -152,7 +151,7 @@ async def test_rollback(ops_test, continuous_writes) -> None:
     new_revisions["x86_64"] = "69"
     with snap_revisions.open("w") as file:
         json.dump(new_revisions, file)
-    charm = await charm_local_build(ops_test)
+    charm = await charm_local_build(ops_test, refresh=True)
```

Comment:
this was causing a `FileNotFound` error. The parameter was incorrectly removed on another PR